pkg/oc/admin/prune/imageprune/prune.go

```diff
@@ -687,23 +712,27 @@ func (p *pruner) addBuildStrategyImageReferencesToGraph(referrer *kapi.ObjectRef
 		}
 		ref, err := imageapi.ParseDockerImageReference(from.Name)
 		if err != nil {
-			msg := fmt.Sprintf("failed to parse DockerImage name %q of %s: %v", from.Name, getKindName(referrer), err)
-			glog.V(4).Infof(msg)
-			return []error{newErrBadReferenceToImage(from.Name, referrer, err.Error())}
+			glog.V(4).Infof("failed to parse DockerImage name %q of %s: %v", from.Name, getKindName(referrer), err)
```

Comment:
can you capitalize this please? also make it warning as above?

Reply:
@mfojtik done